src/cuga/sdk_core/tests/test_supervisor_yaml_config.py

```
@@ -209,3 +209,72 @@ async def test_mcp_servers_config(self):
+  strategy: adaptive
+
+agents:
+  - name: bad_agent
+    import_from: non_existent_module.non_existent_attr
+"""
+        with tempfile.NamedTemporaryFile(mode="w", suffix=".yaml", delete=False) as f:
+            f.write(yaml_content)
+            temp_path = f.name
+
+        try:
+            with pytest.raises(Exception):
```

Ruff (0.15.12)

[warning] Do not assert blind exception: `Exception`

(B017)